prov/verbs/src/verbs_info.c

```diff
@@ -623,7 +623,7 @@ static int fi_ibv_get_device_attrs(struct ibv_context *ctx,
 	}
 
 	if (port_num == device_attr.phys_port_cnt + 1) {
-		VERBS_WARN(FI_LOG_FABRIC, "There are no active ports\n");
+		VERBS_DBG(FI_LOG_FABRIC, "There are no active ports\n");
```

Comment:
We should at least make this INFO level log since on systems with just one verbs device this can be fatal failure.

Reply:
Sounds reasonable, I'll update the PR. Thanks.